jina/parsers/__init__.py

```diff
@@ -17,6 +17,7 @@ def set_pod_parser(parser=None):
 
     from jina.parsers.orchestrate.base import mixin_scalable_deployment_parser
     from jina.parsers.orchestrate.pod import mixin_pod_parser, mixin_hub_pull_options_parser
+    from hubble.executor.parsers.pull import mixin_hub_pull_options_parser
```

Comment:
it is done in purpose not to add this,  so that `instalL_requirements` is redefined by us

Reply:
In the latest SDK I've separated the `install_requirements` just for this reason.
https://github.com/jina-ai/jina-hubble-sdk/pull/150/files

Reply:
I'll also bump the version lock in this PR